### DISCUSSION TITLE
Supporting newer flat format eslint.config.json

### DISCUSSION THREAD

**Discussion by skalidindi** [typescript-eslint/typescript-eslint#7125] [Technical Discussions]
Does this plugin support the newer file format of eslint? If so are there any docs or examples configuring this plugin with the new format?

See https://eslint.org/docs/latest/use/configure/configuration-files-new

**Comment by bradzacher** [comment]
As per our [contributor docs](https://typescript-eslint.io/contributing/discussions) we don't use discussions as a Q&A/support forum.

See existing issues #5908, #5938, #6420, #6282
